client/control/tests/reducers/test_columnReducer.js

```diff
@@ -0,0 +1,129 @@
+import appReducer from 'control/reducers';
+import * as actions from 'control/actions/ColumnActions';
+
+import {
+  initialState,
+} from 'control/tests/fixtures';
+
+/**
+ * Utility function to clone the initialState column array and its values
+ * (This prevents nested objects from being cloned by reference)
+ *
+ * @return {Array} New array with cloned initialState values
+ */
+const cloneInitialArray = () => JSON.parse(JSON.stringify(initialState.columns));
+
+
+/**
+ * Utility function to get the index of a column
+ * which is NOT shown by default.
+ *
+ * @return {Number} Index of non-default column to test against.
+ */
+const getNondefaultColumn = () => {
+  let foundIndex;
+  initialState.columns
+    .find((col, index) => {
+      foundIndex = index;
+      return !col.enabled;
+    });
+
+  if (!foundIndex) {
```

Comment:
Should this compare specifically to undefined, since `0` is a valid value? Also, should this be an exception instead of just printing a warning? Test code probably shouldn't be tolerant of weird cases.